boards: nordic: Fix RRAM size for nRF54l10 NS

Vge0rge opened the pull request:
The nRF54L1 has 1012 KB RRAM and not 1022 KB as
    it was set before. This adjusts the NS target
    with the correct size.

Vge0rge committed:
Revert "boards: nordic: nrf54l10_cpuapp_ns: Add workaround for partitions"

This reverts commit 002838ab9ef526c882225504bda0ef86ff577455.

Signed-off-by: Georgios Vasilakis <georgios.vasilakis@nordicsemi.no>

Vge0rge committed:
boards: nordic: Fix RRAM size for nRF54l10 NS

The nRF54L1 has 1012 KB RRAM and not 1022 KB as
it was set before. This adjusts the NS target
with the correct size.

Signed-off-by: Georgios Vasilakis <georgios.vasilakis@nordicsemi.no>

greg-leach reviewed (commented):
If possible, can the equivalent Ezurio board files be updated in line with the Nordic board changes?

Vge0rge committed:
manifest: Bring TF-M with fixed nRF54L10 NS RRAM size

Corrects the RRAM size in the TF-M repo.

Signed-off-by: Georgios Vasilakis <georgios.vasilakis@nordicsemi.no>

Vge0rge committed:
boards: ezurio: Fix RRAM size for nRF54l10 NS

The nRF54L1 has 1012 KB RRAM and not 1022 KB as
it was set before. This adjusts the NS target
RRAM size for the relevant Ezurio boards
which use the nRF54L10.

Signed-off-by: Georgios Vasilakis <georgios.vasilakis@nordicsemi.no>

Vge0rge commented on boards/ezurio/bl54l15_dvk/bl54l15_dvk_nrf54l10_cpuapp_ns.dts:
I added the change for the Ezurio boards as well yes

degjorva commented:
The comments for the RAM is wrong as well: https://github.com/zephyrproject-rtos/zephyr/blob/e486371174e9dcb8fc4770a691e988988c565afd/boards/ezurio/bl54l15_dvk/bl54l15_dvk_nrf54l10_cpuapp_ns.dts#L35-L36

https://github.com/zephyrproject-rtos/zephyr/blob/e486371174e9dcb8fc4770a691e988988c565afd/boards/nordic/nrf54l15dk/nrf54l15dk_nrf54l10_cpuapp_ns.dts#L34-L35

Should be 72 to match the dts

tomi-font commented:
@rerickson1 please review or remove yourself as assignee

rerickson1 commented:
> @rerickson1 please review or remove yourself as assignee

Sorry, I was waiting for the manifest to be ready. Won't it have to be reviewed again anyway?

Vge0rge commented:
@greg-leach @nordicjm  Please reapprove, the only change is that I fixed a mistake in the comments on the dts files, nothing else.

Vge0rge commented:
Just updated the manifest because the TF-M PR got merged.
